optimize/backend/pom.xml

```diff
@@ -398,20 +398,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+
     <dependency>
-      <groupId>io.camunda</groupId>
-      <artifactId>camunda-service</artifactId>
-      <version>${project.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>io.camunda</groupId>
-          <artifactId>zeebe-scheduler</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>io.camunda</groupId>
-          <artifactId>zeebe-gateway-protocol-impl</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>org.camunda.bpm</groupId>
+      <artifactId>camunda-license-check</artifactId>
+      <version>2.9.0</version>
```

Comment:
This version is accessible for everyone right?

Reply:
Yeah, this is the version that was placed in a public repo